### PR TITLE
Fix TICS report workflow, revert `execCommand` addition, set rate limits

### PR DESCRIPTION
**Pull request by lorumic**
- Reverted the addition of an `execCommand` endpoint to reduce CodeQL warnings.
- Set rate limits to reduce CodeQL warnings.
- Use GH runners instead of self-hosted ones for the TICS workflow.

### CODE REVIEW

File: js/tests/e2e/server.js [hunk 1]
```diff
@@ -32,7 +32,17 @@ const {
 } = require("./fixtures/constants.cjs");
 
 const app = express();
-app.use(express.json());
+
+// set up rate limiter: maximum of ten requests per minute
+var RateLimit = require("express-rate-limit");
+var limiter = RateLimit({
```
Comment (anusha-c18) [inline js/tests/e2e/server.js:38]:
```suggestion
const limiter = rateLimit({
```

File: js/tests/e2e/server.js [hunk 2]
```diff
@@ -32,7 +32,17 @@ const {
...
+  windowMs: 15 * 60 * 1000, // 15 minutes
+  max: 200, // max 200 requests per windowMs
```
Comment (anusha-c18) [inline js/tests/e2e/server.js:40]: should we add a message as well?

**Review by anusha-c18 — CHANGES_REQUESTED** [review]
Looks great already! Left a couple of suggestions

File: js/tests/e2e/server.js (hunk 2 above)
Comment (lorumic) [reply js/tests/e2e/server.js:40]: I think the default "Too many requests" one is fine. I accidentally hit it [before](https://github.com/canonical/anbox-streaming-sdk/actions/runs/12297329997/job/34318480387#step:11:1789), when the rate limit was too restrictive even for our use case :sweat_smile:

File: js/tests/e2e/server.js [hunk 3]
```diff
@@ -32,7 +32,17 @@ const {
 } = require("./fixtures/constants.cjs");
 
 const app = express();
-app.use(express.json());
+
+// set up rate limiter: maximum of ten requests per minute
+var RateLimit = require("express-rate-limit");
```
[the remark this answers is not in the document]
Comment (lorumic) [reply js/tests/e2e/server.js]: Addressed [here](https://github.com/canonical/anbox-streaming-sdk/compare/c738ec2ff20dee023fec04acc43bc051de8239cc..c5204f39a75226e871bb5bf06e2fa9354cdb928b)

File: js/tests/e2e/server.js (hunk 1 above)
Comment (lorumic) [reply js/tests/e2e/server.js]: Addressed [here](https://github.com/canonical/anbox-streaming-sdk/compare/c738ec2ff20dee023fec04acc43bc051de8239cc..c5204f39a75226e871bb5bf06e2fa9354cdb928b)

File: js/tests/e2e/server.js (hunk 2 above)
Comment (anusha-c18) [reply js/tests/e2e/server.js:40]: I see, sounds good 👍

**Review by anusha-c18 — APPROVED** [review]
Thanks for the quick fix! LGTM 👍

**Review by morphis — APPROVED** [review]
LGTM

**Pull request merged**